base/message_loop/message_pump_io_starboard.cc

```diff
@@ -151,9 +205,16 @@ bool MessagePumpIOStarboard::Watch(SbSocket socket,
     SbSocketWaiterRemove(waiter_, old_socket);
   }
 
+  if (!SbSocketIsValid(socket)) {
+    NOTREACHED() << "Invalid socket" << socket;
+    return false;
+  }
+
   // Set current interest mask and waiter for this event.
   if (!SbSocketWaiterAdd(waiter_, socket, controller,
                          OnSocketWaiterNotification, interests, persistent)) {
+    SB_DLOG(ERROR) << __FUNCTION__
+                   << " Failed to remain interested in interests=" << interests;
```

Comment:
This would be a confusing error statement for someone who doesn't know much about sockets, can we rephrase to something like "Failed to register interests: "?